exercises/ocr-numbers/.meta/hints.md

```diff
@@ -0,0 +1,4 @@
+Some editors trim whitespace. If you rely on trailing whitespace in a multiline string, 
```

Comment:
Surely a multiline string comes to mind, but it is not restricted to that.  It could be a single line HereDoc as well.